Langs/0804.lang

```diff
@@ -20,24 +30,25 @@ RAM Installed=内存大小
 Secure Boot=安全启动 (Secure Boot)
 Storage Available=可用磁盘容量
 TPM Version=TPM 版本
-Checking...=正在检查，请稍候……
+Checking...=正在检测，请稍候……
 64 Bit CPU=64 位 CPU
 64 Bit OS=64 位操作系统
 32 Bit CPU=32 位 CPU
 32 Bit OS=32 位操作系统
 Unable to Determine=无法确定
 Unable to Check List=无法验证
 Error Accessing List=读取兼容列表时出错
-Not Currently Listed as Compatible=未在兼容列表中列出
-Listed as Compatible=在兼容列表中
+Not Currently Listed as Compatible=与系统要求不兼容
+Listed as Compatible=符合系统要求
```

Comment:
I dont think translations for line 41 and 42 convey the original meaning.

*Not on the compatible list* does not necessarily mean *incompatible* or *not meeting system requirement*. It is the user's judgment after all.

You can't plain tell the users that their machine does not meet the system requirements, just becauae their CPU is *not listed as compatible*.

I'd suggest keeping the original translation for line 41 to 42.